torch/_subclasses/meta_utils.py

```diff
@@ -556,10 +570,6 @@ def empty_create(inner_t, inner_src):
                                 # format here
                                 r = r.clone(memory_format=torch.preserve_format)
 
-                    # Graph-Break for wrapped tensors
-                    if torch._C._functorch.is_functorch_wrapped_tensor(t):
-                        return NotImplemented
```

Comment:
we should still keep this here. BatchedTensor is not the only Functorch wrapped tensor. Instead we should change the condition to "if not is_batchedtnesor and is_functorch_wrapped_tensor"